src/ruler.js

```diff
@@ -141,9 +141,10 @@ export default class Ruler {
                     cornerStyle = 'rul_corner' + side.toUpperCase();
 
                 corner.title = 'Clear Guide lines';
-                this.utils.addClasss(corner, ['rul_corner', cornerStyle]);
+                this.utils.addClasss(corner, ['rul_corner fa fa-trash', cornerStyle]);
```

Comment:
I think it would be better to add this as an option.
```js
this.utils.addClasss(corner, ['rul_corner', cornerStyle, options.cornerIcon]);
```
The add the default value of `cornerIcon: fa fa-trash` after this line https://github.com/Ju99ernaut/grapesjs-rulers/blob/4638118c1e4898b0dcc00bc8ce1a30d08481b0f5/src/index.js#L29

Reply:
Fixed on new commit